### DISCUSSION TITLE
Gate keeper polices issue's in Opensearch cluster environment

### DISCUSSION THREAD

**Discussion by banbmp** [community/community#69681] [Discussions]
### Select Topic Area

Question

### Body

I disabled the security config in my opensearch values.yaml code, I am getting gatekeeper policy error when I  ran the Helm chart. can anyone tell me what is the safest way to run the security's policy.

find the below error logs.

Enabling execution of install_demo_configuration.sh for OpenSearch Security Plugin
**************************************************************************
** This tool will be deprecated in the next major release of OpenSearch **
** https://github.com/opensearch-project/security/issues/1755           **
**************************************************************************
OpenSearch Security Demo Installer
** Warning: Do not use on production or public reachable systems **
Basedir: /usr/share/opensearch
OpenSearch install type: rpm/deb on NAME="Amazon Linux"
OpenSearch config dir: /usr/share/opensearch/config
OpenSearch config file: /usr/share/opensearch/config/opensearch.yml
OpenSearch bin dir: /usr/share/opensearch/bin
OpenSearch plugins dir: /usr/share/opensearch/plugins
OpenSearch lib dir: /usr/share/opensearch/lib
Detected OpenSearch Version: x-content-2.9.0
Detected OpenSearch Security Version: 2.9.0.0
/usr/share/opensearch/plugins/opensearch-security/tools/install_demo_configuration.sh: line 200: cannot create temp file for here-document: Read-only file system
/usr/share/opensearch/plugins/opensearch-security/tools/install_demo_configuration.sh: line 229: cannot create temp file for here-document: Read-only file system
/usr/share/opensearch/plugins/opensearch-security/tools/install_demo_configuration.sh: line 260: cannot create temp file for here-document: Read-only file system
/usr/share/opensearch/plugins/opensearch-security/tools/install_demo_configuration.sh: line 291: cannot create temp file for here-document: Read-only file system
/usr/share/opensearch/plugins/opensearch-security/tools/install_demo_configuration.sh: line 322: cannot create temp file for here-document: Read-only file system
tee: /usr/share/opensearch/config/kirk.pem: Read-only file system
Enabling OpenSearch Security Plugin
Enabling execution of OPENSEARCH_HOME/bin/opensearch-performance-analyzer/performance-analyzer-agent-cli for OpenSearch Performance Analyzer Plugin
./opensearch-docker-entrypoint.sh: line 61: /usr/share/opensearch/logs/PerformanceAnalyzer.log: Read-only file system
Exception in thread "main" java.nio.file.FileSystemException: /tmp/opensearch-8437471976345703635: Read-only file system
        at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:100)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
        at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
        at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:397)
        at java.base/java.nio.file.Files.createDirectory(Files.java:700)
        at java.base/java.nio.file.TempFileHelper.create(TempFileHelper.java:134)
        at java.base/java.nio.file.TempFileHelper.createTempDirectory(TempFileHelper.java:171)
        at java.base/java.nio.file.Files.createTempDirectory(Files.java:1017)
        at org.opensearch.tools.launchers.Launchers.createTempDirectory(Launchers.java:79)
        at org.opensearch.tools.launchers.TempDirectory.main(TempDirectory.java:67)

**Comment by samus-aran** [comment]
Hi @banbmp ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/opensearch-project/security). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
